Libraries/oneCCL/oneCCL_Getting_Started/sample.json

```diff
@@ -0,0 +1,26 @@
+{
+ "guid": "C56209D9-5CF1-4EEC-AE95-596D81640AEB",
+ "name": "oneCCL Getting Started",
+ "categories": ["Toolkit/Intel® oneAPI Base Toolkit/oneCCL"],
+ "description": "Basic oneCCL programming model for both Intel CPU and GPU.",
+ "toolchain": ["dpcpp"],
+ "languages": [{"cpp":{}}],
+ "dependencies": ["ccl"],
+ "os": ["linux"],
+ "builder": ["cli","cmake"],
+ "targetDevice": ["CPU", "GPU"],
+ "ciTests": {
+	"linux": [{
+		"env": ["source /opt/intel/oneapi/setvars.sh --ccl-configuration=cpu_gpu_dpcpp --force" ],
+		"id": "gsg",
+		"steps": [
+			"mkdir build",
+      		        "cd build",
+           		"cmake .. -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=dpcpp",
+           		"make",
+			"mpirun -n 2 ./out/sycl/sycl_allreduce_cpp_test cpu",
+			"mpirun -n 2 ./out/sycl/sycl_allreduce_cpp_test gpu"
```

Comment:
I think this test should be split into 2 configurations, one for CPU the other one for GPU. Same for all other samples.